push.cpp

```diff
@@ -647,6 +647,31 @@ class CPushMod : public CModule
 
 				PutDebug("payload: " + params["payload"]);
 			}
+            else if (service == "pushjet")
+            {
+                if (options["secret"] == "")
+                {
+                    PutModule("Error: secret (service key) not set");
+                    return;
+                }
+                
+                service_host = "api.pushjet.io";
+                service_url = "/message";
+                
+                params["secret"] = options["secret"];
+                params["title"] = message_title;
+                params["message"] = message_content;
+                
+                if (message_uri != "")
+                {
+                    params["link"] = message_uri;
+                }
+                
+                if (options["message_priority"] != "")
+                {
+                    params["level"] = options["message_priority"];
+                }
+            }
```

Comment:
Can you fix the indenting to use tabs to be consistent with the rest of the file?